mapper.csv

```diff
@@ -0,0 +1,45 @@
+0,0,0
```

Comment:
Can you add a content description to the beginning of this file? This way if we need to refer back to how we rename the runs we have a record. The description can be multi-line, each of which start with a `#`. Your script will skip any line that starts with this character.

In you description, can you also state how the RMSDs were calculated? I'm guessing they are the RMSDs for the starting structure of each run.